### PR TITLE
Python SDK 4.0 Dev Docs Update

### PR DESCRIPTION
**Pull request by jasnoors**
Python SDK 4.0 Dev Docs Update

## Description

Python SDK 4.0 Dev Docs Update

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/document-generation-api/quickstarts/python/index.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+1) In your Downloads folder, find the ZIP file with your credentials: PDFServicesSDK-Python (Extract, Auto-tag)Samples.zip. If you unzip that archive, you will find a folder of samples and the `pdfservices-api-credentials.json` file.
+
+![Samples](./shot5_spc.png)
+
+2) Take the `pdfservices-api-credentials.json` file and place it in a new directory.
+
+3) At the command line, change to the directory you created, and run the following command to install the Python SDK: `pip install pdfservices-sdk`.
+
+![alt](shot7.png)
```
Comment (varkumaradobe) [inline src/pages/overview/document-generation-api/quickstarts/python/index.md:48]: Please update the screenshot, below screenshot show old python version confusing users

image

Comment (varkumaradobe) [reply src/pages/overview/document-generation-api/quickstarts/python/index.md:48]: Applicable in all 4 locations of quickstarts

File: src/pages/overview/pdf-extract-api/gettingstarted.md [hunk 2]
```diff
@@ -593,7 +593,7 @@ After downloading the zip, you can run the samples in the zip directly by settin
 
 #### Setup a Python environment
 
-1. Install [Python 3.6](https://www.python.org/downloads/) or higher.
+1. Install [Python 3.10](https://www.python.org/downloads/) or higher.
```
[the remark this answers is not in the document]
Comment (varkumaradobe) [reply src/pages/overview/pdf-extract-api/gettingstarted.md:596]: Docs level changes will be required under other languages as well for correctness.

**Review by varkumaradobe — CHANGES_REQUESTED** [review]

**Comment by varkumaradobe** [comment]
@jasnoors please add screenshot of logs of a working example

image
image

**Review by hitesh-kumar-mahour — APPROVED** [review]

**Pull request merged**